.travis.yml

```diff
@@ -45,6 +45,8 @@ matrix:
       env: COMPOSER_FLAGS="--prefer-lowest"
     - php: '7.3'
       env: SYMFONY=4.4.*
+    - php: '7.3'
+      env: SYMFONY=5.0.*
```

Comment:
:thinking: :thinking: :thinking: why?

Reply:
Should add this to allow failures?

Reply:
We have no support for symfony 5 for now, to this will always fail

Reply:
shall we remove this on dev-kit?

Reply:
> shall we remove this on dev-kit?

Let's remove it and add it again when symfony 5 is supported (when SonataIntlBundle is removed), I'll make a PR now.